ios/FluentUI/Bottom Sheet/BottomSheetView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class BottomSheetView: UIView {
```

Comment:
Possibly worth a comment explaining why we need this to exist and have a subview? The naive view of this is why do we need a subclass at all and can't we just use contentView directly if it takes our entire and we're not using any UIView subclassing hooks.

So the 3 possible outcomes:
1) Leave the way it is
2) Convert it from a subclass to a factory function to return a view with a subview
3) Convert it from a subclass to a factory function to return just one view.

If it's not the simplest option (#3), and I'm pretty sure there is a good reason it can't be, we should probably add a comment explaining the need for the subview at least.

Reply:
I explain the need for the subview in a comment below, basically if you apply both a shadow and round corners with clipsToBounds == true on the same view, it will clip its own shadow.

Not sure I like the factory approach, because it doesn't clearly connect the two. It would have to return both the root view (which is what externally is the bottom sheet and that's what you place and size in your hierarchy) and the contentView (which is where all the content needs to go. I thought about redirecting all addSubview calls to contentView.addSubview, but that feels kinda wrong. Or is it? Is there an obvious solution here I'm missing?

Reply:
Does the contentView actually need to be returned? Just make it something like a 
```
private func makeBottomSheetByEmbedding(view:  UIView) -> UIView
```
where it takes in the view that needs to be placed into contentView and does it for you

Reply:
ah, so there was an obvious solution I was missing :D